### PR TITLE
Updated doc's landing page's title and subtitle

### PR DESCRIPTION
**Pull request by pawartur**

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula')
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'js-did',
-  tagline: 'Decentralized Identifiers tools',
+  title: 'Decentralized Identifiers (DIDs)',
```
Comment (oed) [inline website/docusaurus.config.js:9]: this also populates the top left title?

**Review by oed — APPROVED** [review]
ty!

File: website/docusaurus.config.js (hunk 1 above)
Comment (pawartur) [reply website/docusaurus.config.js:9]: It doesn't, @oed . Do you want me to put 'DIDs' in top left title? I'm thinking that 'Decentralized Identifiers (DIDs)' may be a bit too long there.

Comment (oed) [reply website/docusaurus.config.js:9]: Ok, not sure how it would look. Whatever you think is best.

Comment (pawartur) [reply website/docusaurus.config.js:9]: Put 'Decentralized Identifiers (DIDs)', we can always change it, if we feel it's too long there. Thanks, @oed :)!

**Pull request merged**